src/python/zquantum/core/utils.py

```diff
@@ -433,6 +433,7 @@ def create_object(specs, **kwargs):
     Returns:
         object: object of any type
     """
+    specs = copy.copy(specs)
```

Comment:
Wait why are we removing this line? Without it, you can only "create" specs once which is a strange behavior.